doc/api/assert.md

```diff
@@ -814,6 +831,24 @@ undefined, a default error message is assigned. If the `message` parameter is an
 instance of an [`Error`][] then it will be thrown instead of the
 [`AssertionError`][].
 
+## `assert.mustSettle(asyncfn)
+* `asyncfn` {Promise}
```

Comment:
```suggestion
<!-- YAML
added: REPLACEME
-->
* `asyncfn` {Promise}
```

Reply:
I believe we always have a blank line after the end of the YAML matter, so maybe this?:

```suggestion
<!-- YAML
added: REPLACEME
-->

* `asyncfn` {Promise}
```